contrib/emicklei/go-restful/option.go

```diff
@@ -60,3 +64,19 @@ func WithAnalyticsRate(rate float64) Option {
 		}
 	}
 }
+
+// WithHeaderTags enables the integration to attach HTTP request headers as span tags.
+// Warning: using this feature can risk exposing sensitive data such as authorization tokens
+// to Datadog.
```

Comment:
From the RFC:

> In each integration docs MUST be made clear that for special headers we don't sub-select
> values. E.g. Cookie header can appear multiple times for different cookies that have to be set.
> The cookie name appears in the value.
> We DO NOT provide a way to only select a specific cookie. If a user activates the header
> Cookie then ALL cookies will be transmitted.

So let's be sure to document that in every integration. You can check out the implementation from other languages to see the wording they used if you'd like.

Reply:
Let's also note that setting the integration-specific header tags option in this way will overwrite any existing global header tags configurations that have previously been set.